src/main/java/tech/jhipster/lite/project/infrastructure/secondary/FileSystemProjectsRepository.java

```diff
@@ -26,18 +27,24 @@ class FileSystemProjectsRepository implements ProjectsRepository {
   private static final String HISTORY_FOLDER = ".jhipster/modules";
   private static final String HISTORY_FILE = "history.json";
   private static final String PRESET_FOLDER = "/";
-  private static final String PRESET_FILE = "preset.json";
 
   private final ObjectMapper json;
   private final ProjectFormatter formatter;
   private final ProjectFiles projectFiles;
   private final ObjectWriter writer;
   private final FileSystemProjectDownloader downloader;
-
-  public FileSystemProjectsRepository(ObjectMapper json, ProjectFormatter formatter, ProjectFiles projectFiles) {
+  private final String presetFile;
```

Comment:
I think it should use

```suggestion
  private final PresetName presetName;
```